### ISSUE TITLE
Multi-monitor

### ISSUE THREAD

**Issue by Annhilate** [Sneezry/authenticator#51]
App does not show when used on any display other than main. It's not a pop up issue because it works on the main display.

**Comment by Sneezry** [comment]
Hi @Annhilate , do you mean the extension doesn't show the pop up window when you click the icon if Chrome is not in your main display?

**Comment by Sneezry** [comment]
@Annhilate 
It seems a bug of Chrome, see https://code.google.com/p/chromium/issues/detail?id=408413

A solution is given at # 10, you may have a try:

Chrome Menu > Settings > Show advanced settings > System.
Use hardware acceleration when available > Uncheck this option and restart Chrome.

**Comment by Annhilate** [comment]
That is correct...when you click the add-in icon in the top right corner nothing happens.
 

--

Dave "The most beautiful thing we can experience is the mysterious. It is the source of all true art and science. He to whom this emotion is a stranger, who can no longer pause to wonder and stand rapt in awe, is as good as dead: his eyes are closed." - Albert Einstein 

    On Thursday, February 18, 2016 1:00 AM, Li Zhe <notifications@github.com> wrote:
 

 @Annhilate 
It seems a bug of Chrome, see https://code.google.com/p/chromium/issues/detail?id=408413A solution is given at # 10, you may have a try:Chrome Menu > Settings > Show advanced settings > System.
Use hardware acceleration when available > Uncheck this option and restart Chrome.—
